tests/ut_mdesktopentry.cpp

```diff
@@ -6,6 +6,12 @@
 
 #include "mdesktopentry.h"
 
+#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
+#define ENDL endl
+#else
+#define ENDL Qt::endl
+#endif
```

Comment:
Let's just use std::endl and avoid this whole qt version problem.

Reply:
You can also do `using endl = Qt::endl` if QTextStream is available and avoid the other changes too.